REPs/REP-33.md

```diff
@@ -17,48 +17,53 @@ Discussions: https://forum.rss3.io/t/proposal-on-node-state-transition/173
-7. **Online** → (current Epoch) → **Slashed**: An `Online` Node transitions to `Slashed` state immediately within the same epoch if it is slashed.
-8. **Slashed** → (next Epoch) → **Online**: A `Slashed` Node transitions to `Online` state in the next epoch if the Operator manually re-online it by the end of the current epoch.
-9. **Offline** → (next Epoch) → **Online**: An `Offline` Node transitions to `Online` state in the next epoch if the Operator manually re-online it by the end of the current epoch.
-10. **Offline** →(after 30 Epochs) → **Exited**: An `Offline` Node transitions to `Exited` state after 30 Epochs of inactivity.
-11. **Exited** → (anytime) → **Registered**: An `Exited` Node can re-register at any time.
+1. **None** → **Registered**: A `None` Node transitions to `Registered` state upon meeting the minimum deposit requirement.
+2. **Registered** → **Initializing**: A `Registered` Node transitions to the `Initializing` state when it starts and begins indexing data but has not yet completed the process.
+3. **Registered** → **Outdated**: A `Registered` Node transitions to the `Outdated` state when the Node is started and its version does not meet the minimum requirement.
+4. **Registered**, **Outdated**, **Initializing**, **Slashed** → (anytime) → **Exited**: When a Node is in any of the `Registered`, `Outdated`, `Initializing`, or `Slashed` states, if the operator chooses to exit, the Node immediately enters the `Exited` state.
+5. **Initializing** → (next Epoch) → **Online**: After the automatic initialization is completed, the Node enters the `Online` state at the next epoch.
+6. **Online** → (current Epoch) → **Exiting**: An `Online` Node transitions to `Exiting` state upon announcing its intention to exit.
+7. **Exiting** → (waiting period) → **Exited**: An `Exiting` Node transitions to `Exited` state after completing the waiting period.
+8. **Online**, **Exiting** → **Slashing**: An `Online` or `Exiting` Node transitions to `Slashing` state if its demotion count reaches the threshold. The operator can appeal within 3 epochs.
+9. **Online**, **Exiting** → (current Epoch) → **Offline**: An `Online` or `Exiting` Node transitions to `Offline` state immediately within the same epoch if the Node is detected to be unavailable.
+10. **Offline**, **Slashed** → (current Epoch) → **Initializing**: An `Offline` or `Slashed` Node transitions to `Initializing` state if the Operator manually brings it back online.
+11. **Slashing** → (after 3 epochs) → **Slashed**: A `Slashing` Node transitions to `Slashed` state after 3 epochs.
+12. **Exited** → (anytime) → **Registered**: An `Exited` Node can re-register at any time.
+
+### Reward Mechanism
+
+Nodes in `Online`, `Initializing`, or `Exiting` states are eligible to provide services and receive rewards. Nodes in other states are ineligible.
```

Comment:
missing `slashing` state?

Reply:
yes, `slashing` state should be rewarded.